src/csm_testing/tests/sat_functional/bootprep/test_bootprep.py

```diff
@@ -600,7 +602,9 @@ def delete_all_cfs_configurations_matching_prefix(cls):
                     configs_json = json.loads(proc.stdout.decode())
 
                     for config in configs_json['configurations']:
-                        if config['name'].startswith(cls.test_prefix):
+                        if config is not None and config[\
+                                'name'].startswith(
+                                cls.test_prefix):
```

Comment:
why the weird line continuation?

```suggestion
                        if config is not None and config['name'].startswith(cls.test_prefix):
```